tests/target-browser/target_browser_test.go

```diff
@@ -535,42 +539,46 @@ var _ = Describe("Target Browser Tests", func() {
 				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("flag needs an argument: %s", flagOperationScope)))
 			})
 
-			It(fmt.Sprintf("Should succeed if flag %s is not provided", flagOperationScope), func() {
+			It(fmt.Sprintf("Should get both OperationScoped backupPlan and clusterBackupPlans if flag %s is not provided",
+				flagOperationScope), func() {
 				args := []string{cmdGet, cmdBackupPlan}
-				args = append(args, commonArgs...)
-				cmd := exec.Command(targetBrowserBinaryFilePath, args...)
-				_, err := cmd.CombinedOutput()
-				Expect(err).ShouldNot(HaveOccurred())
+				backupPlanData := runCmdBackupPlan(args)
+				Expect(len(backupPlanData)).To(Equal(noOfBackupPlansToCreate * 2))
 			})
 
-			It(fmt.Sprintf("Should succeed if flag %s is given zero value", flagOperationScope), func() {
+			It(fmt.Sprintf("Should get both backupPlans and clusterbackupPlans if flag %s is given zero value", flagOperationScope), func() {
 				args := []string{cmdGet, cmdBackupPlan, flagOperationScope, ""}
-				args = append(args, commonArgs...)
-				cmd := exec.Command(targetBrowserBinaryFilePath, args...)
-				_, err := cmd.CombinedOutput()
-				Expect(err).ShouldNot(HaveOccurred())
+				backupPlanData := runCmdBackupPlan(args)
+				Expect(len(backupPlanData)).To(Equal(noOfBackupPlansToCreate * 2))
 			})
 
 			It(fmt.Sprintf("Should get %d backupPlan if flag %s is given %s value", noOfBackupPlansToCreate,
 				flagOperationScope, internal.MultiNamespace), func() {
 				args := []string{cmdGet, cmdBackupPlan, flagOperationScope, internal.MultiNamespace}
 				backupPlanData := runCmdBackupPlan(args)
 				Expect(len(backupPlanData)).To(Equal(noOfBackupPlansToCreate))
+				for idx := range backupPlanData {
+					Expect(backupPlanData[idx].Kind).To(Equal(internal.ClusterBackupPlanKind))
+				}
 			})
 
 			It(fmt.Sprintf("Should get one backupPlan for specific clusterBackupPlan UID with flag %s", cmd.OperationScopeFlag), func() {
-				args := []string{cmdGet, cmdBackupPlan, clusterBackupPlanUIDs[1], flagOperationScope, internal.MultiNamespace}
+				args := []string{cmdGet, cmdBackupPlan, backupPlanUIDs[1], flagOperationScope, internal.MultiNamespace}
```

Comment:
How do we know whether `backupPlanUIDs[1]` is backupPlan's UID or ClusterBackupPlan's UID?

Reply:
Here I had created the only clusterBackupPlan. Now added both